src/main/java/org/littil/api/mail/MailService.java

```diff
@@ -33,6 +37,16 @@ public void sendContactMail(String recipientEmailAddress, String contactMessage,
         send(template);
     }
 
+    public void sendFeedbackMail(FeedbackPostResource feedback, Optional<String> feedbackEmail) {
+        if (feedbackEmail.isEmpty()) {
+            return;
```

Comment:
but in any case; log.error that no feedback email is set and feedback is ignored (with values)

Reply:
log warning when email address is not set